### ISSUE TITLE
create a simple form for collecting meme

### ISSUE THREAD

**Issue by xeon-zolt** [ILUGD/ILUGD.github.io#68]
idea
a simple bootstrap form that will collect meme upload it to Imgur using API(private upload not public) then creating a PR updating JSON from the data collected from the form 
if anyone has other suggestion to create this pipeline please suggest we don't have any servers for hosting keep that in mind

**Comment by cvam0000** [comment]
good idea but how without a server ? my suggestion is to make a telegram bot that will update the json 
if there is any entry by admin in the meme group ??

**Comment by xeon-zolt** [comment]
that also works much better solution mine might have created too much spam PR

**Comment by xeon-zolt** [comment]
server in my solution is Heroku which don't provide DB but can always upload to Imgur but how can we make it easier for anyone to send a meme

**Comment by cvam0000** [comment]
oh yes, Free Servers on Heroku :)
